.github/CODEOWNERS

```diff
@@ -42,6 +42,7 @@ sycl/source/detail/pi.cpp @smaslov-intel
 sycl/source/detail/plugin.hpp @smaslov-intel
 sycl/source/detail/posix_pi.cpp @smaslov-intel
 sycl/source/detail/windows_pi.cpp @smaslov-intel
+sycl/plugins/esimd_cpu/ @kbobrovs
```

Comment:
```suggestion

# ESIMD CPU emulator plug-in
sycl/plugins/esimd_cpu/ @kbobrovs
```

Konst, can we add one more code owner to back you up?

Reply:
Good idea. @smaslov-intel - would you be willing to be the back-up owner for this?

Reply:
OK, please add me.